test/odbc/psqlodbc/psqlodbc_tests_common.cpp

```diff
@@ -630,6 +630,50 @@ void testComparisonFunctions(ServerType serverType, const string &tableName, con
     EXPECT_EQ(string(colResults[i]), expectedResults[i]);
   }
 
+  // Assert that there is no more data
+  rcode = SQLFetch(odbcHandler.GetStatementHandle());
+  EXPECT_EQ(rcode, SQL_NO_DATA);
+  odbcHandler.CloseStmt();
+}
+
+void testStringFunctions(ServerType serverType, const string &tableName, const vector<string> &operationsQuery, 
```

Comment:
I can't tell what this is doing from the name. Please have a better name for this along with documentation in the .h file.